api/src/main/java/org/openmrs/module/fhir2/api/FhirMedicationRequestService.java

```diff
@@ -28,5 +28,5 @@ public interface FhirMedicationRequestService extends FhirService<MedicationRequ
 	IBundleProvider searchForMedicationRequests(ReferenceAndListParam patientReference,
 	        ReferenceAndListParam encounterReference, TokenAndListParam code, ReferenceAndListParam participantReference,
 	        ReferenceAndListParam medicationReference, TokenAndListParam id, DateRangeParam lastUpdated,
-	        HashSet<Include> includes);
```

Comment:
I've changed the signature of this service method... is this okay (ie is this service expected to be used anywhere outside this module)?  If not, easy enough to create a new service method with an additional parameter as well.

Reply:
The general rule is that these service classes are not general service classes. For the most part they are supposed to the the things called by the ResourceProviders. Basically, the ResourceProviders provide the FHIR REST API and the service classes translate that into other calls. As long as the functionality still works, it should be fine to change the signatures like this.